pkg/controller/propagator/propagation.go

```diff
@@ -676,22 +685,37 @@ func (r *ReconcilePolicy) processTemplates(replicatedPlc *policiesv1.Policy, dec
 			reqLogger.Error(tplErr, "Failed to resolve templates")
 
 			r.recorder.Event(rootPlc, "Warning", "PolicyPropagation",
-				fmt.Sprintf("Failed to resolve templates for policy %s/%s for cluster %s/%s .", rootPlc.GetName(), rootPlc.GetNamespace(), decision.ClusterNamespace, decision.ClusterName))
+				fmt.Sprintf("Failed to resolve templates for cluster %s/%s: %s", decision.ClusterNamespace, decision.ClusterName, tplErr.Error()))
```

Comment:
```suggestion
				fmt.Sprintf("Failed to resolve templates for cluster %s/%s: %v", decision.ClusterNamespace, decision.ClusterName, tplErr))
```

Reply:
previously we had just a generic msg,  added a msg so its a bit more clear without having to go to logs.

I was hoping to  keep this short ,  just have the main msg which is already quite big. ,  because its on an "event",  

do I want to the put the object in there ?  can you  tell why you are recommending that change ?

Reply:
It's the same result, it's just customary to pass the error with format directive `%v`, which automatically uses the string returned from `tmpl.Error()`. You can see examples of this:
https://go.dev/blog/error-handling-and-go

It just makes the code shorter and is a more customary approach, that's all.

Reply:
isnt it results in the same , I'd to leave it the way it is.